pkg/kubevirt/core/core.go

```diff
@@ -160,7 +147,9 @@ func (p PluginSPIImpl) CreateMachine(ctx context.Context, machineName string, pr
 					"ReadWriteOnce",
 				},
 				Resources: corev1.ResourceRequirements{
-					Requests: pvcRequest,
+					Requests: corev1.ResourceList{
+						corev1.ResourceStorage: *providerSpec.PVCSize,
```

Comment:
Potential nil pointer panic: `providerSpec.PVCSize` could be nil. Can be avoided by using a non-pointer type.